internal/loader/vsp.go

```diff
@@ -4,27 +4,27 @@ import (
 	"sync"
 
 	"decred.org/dcrwallet/v3/errors"
-	"decred.org/dcrwallet/v3/internal/vsp"
+	vsp "github.com/decred/vspd/client/v2"
 )
 
 var vspClients = struct {
 	mu      sync.Mutex
-	clients map[string]*vsp.Client
+	clients map[string]*vsp.AutoClient
 }{
-	clients: make(map[string]*vsp.Client),
+	clients: make(map[string]*vsp.AutoClient),
 }
 
 // VSP loads or creates a package-global instance of the VSP client for a host.
 // This allows clients to be created and reused across various subsystems.
-func VSP(cfg vsp.Config) (*vsp.Client, error) {
+func VSP(cfg vsp.Config) (*vsp.AutoClient, error) {
 	key := cfg.URL
 	vspClients.mu.Lock()
 	defer vspClients.mu.Unlock()
 	client, ok := vspClients.clients[key]
 	if ok {
 		return client, nil
 	}
-	client, err := vsp.New(cfg)
+	client, err := vsp.New(cfg, log)
```

Comment:
This is problematic because it is currently passing the logger for the `LODR` subsystem. @jrick could you advise how you would prefer to fix this?

Reply:
Not totally sure what you mean by this.

Do you mean all of the loggers in every internal package should be moved into a new internal/loggers package? Just the loader/vsp one?
If they are moved to a seperate interal package, I guess they need to be set in the top level `log.go` and then exported so they can be accessed in other internal packages?

Reply:
decred/dcrwallet#2226